index.html

```diff
@@ -116,5 +116,28 @@ <h3 class="my-work-title">Multi-Post Stories</h3>
       <button class="visualize-button" type="button">See Project</button>
   </div>
 </section>  
+<form class="my-contact-section" action="https://formspree.io/f/mpzbgdrg" method="POST">
+    <h2 class="contact-us">CONTACT US</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+      <ul class="my-form">
+          <li>
+              <input type="text" name="user_name" placeholder="Text input" maxlength="30" required>
+          </li>
+          <li>
+              <input type="email" name="user_email" placeholder="yourname@example.com" required>
+          </li>
+          <li>
+             <textarea name="user_name" id="msg" maxlength="500" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Please make sure that the `textarea` as well is `required`, because all the input fields are supposed to be required.  😁